src/main/kotlin/app/services/impl/S3ObjectServiceImpl.kt

```diff
@@ -41,7 +41,7 @@ class S3ObjectServiceImpl(private val amazonS3: AmazonS3, private val retriedBat
                addUserMetadata("iv", encryptingOutputStream.initialisationVector)
                addUserMetadata("cipherText", encryptingOutputStream.dataKeyResult.ciphertextDataKey)
                addUserMetadata("dataKeyEncryptionKeyId", encryptingOutputStream.dataKeyResult.dataKeyEncryptionKeyId)
-               addUserMetadata("data_product", topicName)
+               addUserMetadata("data_product", topicName.replace("db.", ""))
```

Comment:
Does this not have the potential to break other data products???

Reply:
No other data product use this metadata as per my understanding, but worth checking with others.